ecosystem/indexer-grpc/indexer-grpc-fullnode/src/stream_coordinator.rs

```diff
@@ -187,9 +187,21 @@ impl IndexerStreamCoordinator {
         // Stage 3: send responses to stream
         let sending_start_time = std::time::Instant::now();
         for response in responses {
-            if self.transactions_sender.send(Ok(response)).await.is_err() {
-                // Error from closed channel. This means the client has disconnected.
-                return vec![];
+            match self.transactions_sender.send(Ok(response)) {
+                Ok(_) => (),
+                Err(e) => {
+                    // Error from closed channel. This means the client has disconnected.
+                    error!(
+                        start_version = first_version,
+                        end_version = end_version,
+                        num_transaction,
+                        highest_known_version,
+                        duration_in_secs = sending_start_time.elapsed().as_secs_f64(),
+                        error = format!("{:?}", e),
+                        "[Indexer Fullnode] Error sending transactions to client."
+                    );
```

Comment:
just log no metrics?

Reply:
fullnode latest version metric caught this error already